tests/test_restore_names.py

```diff
@@ -1,3 +1,28 @@
 from app.restore_names import restore_names
 
-# write your tests here
+
+def test_the_first_name_is_none_or_key_not_found():
```

Comment:
Could you create separate tests for both cases?
It'd be easier to find out what part of function logic is wrong in case of bugs